### DISCUSSION TITLE
Full Export as jpg/png

### DISCUSSION THREAD

**Discussion by BobSanScans** [xulihang/ImageTrans-docs#464] [Ideas]
_Once again, if this was already pointed out in the documentation, I apologize for missing it._

Currently in order to export each page as it's own translated image, it seems you are able to do so individually at 'export current picture as'

Would it be possible to add this feature to the general 'export' tab, so that it will export every image in the project translated as it's own file, instead of having to manually export each page individually?

**Comment by xulihang** [comment]
You can use the generate translated pictures for all in Project-Batch.

![image](https://github.com/xulihang/ImageTrans-docs/assets/5462205/03888d51-6e1c-4dba-b1c5-bd9ed9ba2e54)
